release/serve_tests/workloads/benchmark_utils.py

```diff
@@ -0,0 +1,110 @@
+import time
+import numpy as np
+
+import ray
+
+
+@ray.remote
+class DeploymentHandleClient:
+    def __init__(self, async_func):
+        self.async_func = async_func
+
+    def ready(self):
+        return "ok"
+
+    async def run(self, *args, **kwargs):
+        return await self.async_func(*args, **kwargs)
+
+
+async def measure_latency_ms(async_fn, args, expected_output, num_requests=10):
+    # warmup for 1sec
+    start = time.time()
+    while time.time() - start < 1:
+        await async_fn(args)
+
+    latency_stats = []
+    for _ in range(num_requests):
+        start = time.time()
+        await async_fn(args) == expected_output
+        end = time.time()
+        latency_stats.append((end - start) * 1000)
+
+    return latency_stats
+
+
+async def measure_throughput_tps(async_fn, args, expected_output, duration_secs=10):
+    # warmup for 1sec
+    start = time.time()
+    while time.time() - start < 1:
+        await async_fn(args)
+
+    tps_stats = []
+    for _ in range(duration_secs):
+        start = time.time()
+        request_completed = 0
+        while time.time() - start < 1:
+            await async_fn(args) == expected_output
+            request_completed += 1
+        tps_stats.append(request_completed)
+
+    return tps_stats
+
+
+async def benchmark_throughput_tps(
+    dag_handle,
+    expected,
+    duration_secs=10,
+    num_clients=1,
+):
+    """Call deployment handle in a blocking for loop from multiple clients."""
+    clients = [
+        DeploymentHandleClient.remote(measure_throughput_tps)
```

Comment:
looks like the `DeploymentHandleClient` is just a thin wrapper around `measure_throughput_tps`... why not just use a remote task instead? should be essentially the same